### ISSUE TITLE
User friendly cull management (Feature Request for several modifications)

### ISSUE THREAD

**Issue by landoskape** [cortex-lab/alyx#809]
## Problem
Culling subjects using the Alyx Subject Admin page is a little trickier than it has to be. 

## Standard Workflow
1. Open up one tab on the `Subjects` page, and another tab on the `Cull_subjects` page (both accessible as Common options from Home). 
2. On both pages, switch to `responsible user = "all"` and `line = <my_mouse_line>`.
3. Decide which mice to cull based on their zygosity, which is only visible on the `Subjects` page. 
4. Move to the `Cull_subjects` page, carefully find the row with the same mouses nickname, and then carefully track along the row to check the `To_be_culled` checkbox.
5. Repeat steps 3 and 4 for each mouse very carefully, sometimes for 10s of mice.

## Suggestions
It would be easier (and safer!) to do this if the `Cull_subjects` page was changed in the following way: 
- Had the same default ordering as the `Subjects` page (e.g. first by birthdate, second by nickname)
- Included a column indicating the zygosity of each subject
- The `To_be_culled` column was next to the nickname, to avoid errors in clicking the wrong checkbox

## Notes 
- The first bullet point is not really necessary because Users can choose the ordering once they are on the page, but at least in my opinion the workflow would be more straightforward and efficient if all Subject Admin pages had the same default ordering. 
- Also, it would be amazing if the color of the nickname changed depending on whether the `To_be_culled` checkbox or the `Reduced` checkbox was checked... that's cosmetic but given the fact that culling the wrong subject really sucks, this would be a sweet feature.

**Comment by landoskape** [comment]
Thanks for the help and let me know if I can be of assistance (I am not familiar with django but happy to learn if someone directs me to the right places in the repository).

**Comment by mayofaulkner** [comment]
Hello,

I've added the following feature improvements which are now released on the cortex lab alyx. 
1. Sort cull subjects page in same way as subjects page (name, birth date)
2. Add zygosity filter to cull subjects page
3. Add zygosity column in cull subjects inline view
4. Move to be culled column next to subject nickname on cull subjects page.

Adding in the colours on selected the to_be_culled proved more difficult. It is straightforward to add this after the list has been saved but not before the selection has been submitted.
